docs/UserGuide.md

```diff
@@ -436,11 +423,9 @@ Format: `delete INDEX`
 
 * Deletes the student at the specified `INDEX`.
 
-* The index must be a valid index number of a student shown in the displayed list.
-
 Examples:
 * `list` followed by `delete 2` deletes the 2nd student in TAB.
-* `find n/Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
+* `find n/Betsy` followed by `ldelete 1 1` deletes the 1st lesson for the 1st student in the results of the `find` command.
```

Comment:
This is `delete` not `ldelete` right?

Reply:
i tot this is saying `find n/Betsy` filters the list of students then `ldelete` acts on the 1st student 's 1st lesson in filtered list

Reply:
But this example is under the "Deleting a student" section though, maybe should go under "Find" section instead?

Reply:
oh right, j saw a duplicate eg under `ldelete` section